src/lib/supercomputer.ts

```diff
@@ -58,50 +57,11 @@ export class Supercomputer {
     }, {});
   }
 
-  /**
-   * Flattens config entries.
-   */
-  private flattenConfigValues(config: Config) {
```

Comment:
It seemed a pretty dangerous thing (flattening of config) because would lead to issues once two models will have the same config prop name. With the updated implementation all model configs are absolutely separated from each other.